velox/docs/develop/testing/fuzzer.rst

```diff
@@ -30,7 +30,10 @@ random input vectors.
 
 The Aggregation Fuzzer tests global aggregations (no grouping keys), group-by
 aggregations (one or more grouping keys), distinct aggregations(no aggregates),
-aggregations with and without masks.
+aggregations with and without masks, aggregations over sorted and distinct inputs.
+
+The Aggregation Fuzzer includes testing of spilling and abandoning partial
+aggregation.
 
 The results of aggregations using functions supported by DuckDB are compared
 with DuckDB results.
```

Comment:
Can you also add: Aggergation Fuzzer also supports running queries against a local Presto instance and comparing the results.

Reply:
I thought this would be a separate PR as it requires explaining how to set this up.